Gruntfile.js

```diff
@@ -135,11 +135,10 @@ grunt.initConfig({
 		}
 	},
 
-	// Remove the requireSpacesInsideParentheses override once everything is fixed
 	jscs: {
 		all: {
 			options: {
-				requireCapitalizedComments: null
+				config: ".jscsrc"
```

Comment:
Is this actually needed? If there's a bug with maxErrors we should report it and override maxErrors, but keep letting it use its default config file finding.